glue/sales_val_flagging.py

```diff
@@ -554,6 +554,7 @@ def to_float(value):
         "run_filter",
         "iso_forest_cols",
         "stat_groups",
+        "date_range",
```

Comment:
**[Thought, non-blocking]** It might be nice for this param to be called `time_frame` so that it matches the value in `inputs.yml` exactly, but I don't feel super strongly about it -- `date_range` is clear in spite of being slightly different!

Reply:
Good call, fixed.